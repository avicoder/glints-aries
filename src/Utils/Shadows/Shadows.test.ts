import { Shadows } from './index';

describe('Shadows', () => {
  it('should match snapshot', () => {
    expect(Shadows).toMatchSnapshot();
  });
});
