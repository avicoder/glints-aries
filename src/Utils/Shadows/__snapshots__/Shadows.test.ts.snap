// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shadows should match snapshot 1`] = `
Object {
  "1": Object {
    "down": "0 2px 4px 0 rgba(0, 0, 0, 0.15)",
  },
  "2": Object {
    "down": "0 4px 8px 0 rgba(0, 0, 0, 0.15)",
  },
  "3": Object {
    "down": "0 6px 12px 0 rgba(0, 0, 0, 0.10)",
  },
  "4": Object {
    "down": "0 12px 24px 0 rgba(0, 0, 0, 0.10)",
    "up": "0 -6px 12px 0 rgba(0, 0, 0, 0.10)",
  },
}
`;
